gcp/modules/cluster/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "cluster_name" {
+  type    = string
+}
+
+variable "region" {
+  type    = string
+}
+
+variable "cluster_network" {
+  type    = string
+}
+
+variable "enable_private_endpoint" {
+  type    = bool
+}
+
+variable "enable_private_nodes" {
+  type    = bool
+}
```

Comment:
Are there any sane defaults for these?

Reply:
Not really, it depends on the situation. You can set defaults on the level that calls the module if needed.